tests/regression/test_auxiliary_dm.py

```diff
@@ -28,7 +28,7 @@ def mesh(self):
 
     def bcs(self, Z):
         bcs = [DirichletBC(Z.sub(0), self.analytical_solution(Z.ufl_domain()), "on_boundary"),
-               DirichletBC(Z.sub(1), Constant((0, 0)), "on_boundary")]
+               DirichletBC(Z.sub(1), zero(), "on_boundary")]
```

Comment:
This kind of things were wrong. We're assigning to an RT space here, so dof-wise we only have one value, not two. If someone had written `Constant((0, 1))` they would have seen something completely wrong. Now that latter is an error.